src/DynamoCoreWpf/Views/Search/LibraryContainerView.xaml.cs

```diff
@@ -42,7 +42,6 @@ public SearchView(SearchViewModel searchViewModel, DynamoViewModel dynamoViewMod
             viewModel = searchViewModel;
             this.dynamoViewModel = dynamoViewModel;
 
-            DataContext = viewModel;
```

Comment:
Or we can leave everything as it is now: with quick fix and without binding errors.